pkg/util/mount/mount_windows.go

```diff
@@ -74,8 +74,24 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 			return nil
 		}
 
-		// empty implementation for mounting azure file
-		return os.MkdirAll(target, 0755)
+		// currently only SMB mount is supported
```

Comment:
exit out and log an error if fstype is not smb.

Reply:
fixed, actually it should be "cifs":
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/azure_file/azure_file.go#L232